sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/Azure.Monitor.AspNetCoreTestApp/Azure.Monitor.AspNetCoreTestApp.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Is the plan to use same project for HttpClient test app?

Reply:
Yes. Vishwesh and I have also discussed other scenarios that would benefit from an aspnetcore test app.